java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -265,33 +433,86 @@ public HostColumnVector copyToHost() {
       HostMemoryBuffer hostDataBuffer = null;
```

Comment:
This is not part of this change as it was present from earlier changes. Let me know if that works, @razajafri .

Reply:
I know its not a part of your change. The change is too small of a nit to warrant its own PR. Its not a big deal I don't want to hold the PR for this, but if you make a change it will be nice to see this